.gitignore

```diff
@@ -1,2 +1,6 @@
 *.log
 tmp
+model.signed.yaml
+pc.img
+pc-gadget
+seed.manifest
```

Comment:
To better know where these come from, I suggest commenting or adding them in a `.gitignore` under the corresponding directory.